tests/e2e/helpers/database.ts

```diff
@@ -29,7 +29,7 @@ export async function addNewStandaloneDatabase(databaseParameters: AddNewDatabas
     //Click for saving
     await t.click(addRedisDatabasePage.addRedisDatabaseButton);
     //Wait for database to be exist
-    await t.expect(myRedisDatabasePage.dbNameList.withExactText(databaseParameters.databaseName).exists).ok('The existence of the database', { timeout: 10000 });
+    await t.expect(myRedisDatabasePage.dbNameList.withExactText(databaseParameters.databaseName ?? '').exists).ok('The existence of the database', { timeout: 10000 });
```

Comment:
Can you please explain the `?? ''`? I'm not familiar this

Reply:
It's a Nullish coalescing operator (??)
enables us to specify a fallback for when a value is undefined or null.

it is needed here because databaseParameters has AddNewDatabaseParameters type which parameter databaseName is optional